src/main/java/org/folio/repository/DbUtil.java

```diff
@@ -12,75 +13,23 @@
 
 import java.io.IOException;
 import java.util.Optional;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.CompletionStage;
-import java.util.function.BiFunction;
 
-import io.vertx.core.AsyncResult;
-import io.vertx.core.Promise;
-import io.vertx.core.Vertx;
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
-import io.vertx.core.logging.Logger;
-import io.vertx.core.logging.LoggerFactory;
-import io.vertx.ext.sql.SQLConnection;
 
-import org.apache.commons.lang3.mutable.MutableObject;
-
-import org.folio.common.FutureUtils;
 import org.folio.rest.persist.PostgresClient;
 import org.folio.rest.tools.utils.ObjectMapperTool;
 
 public class DbUtil {
   private DbUtil() {}
 
-  private static final Logger LOG = LoggerFactory.getLogger(DbUtil.class);
-
   public static JsonArray createInsertOrUpdateParameters(String id, String name) {
     return new JsonArray()
       .add(id)
       .add(name)
       .add(name);
   }
 
-  public static <T> CompletableFuture<T> executeInTransaction(String tenantId, Vertx vertx,
```

Comment:
These changes are related to moving executeInTransaction to folio-service-tools repository